### PR TITLE
Split `--workspace` test pipeline to separate jobs

### PR DESCRIPTION
**Pull request by bkontur**
- [X] Does not require a CHANGELOG entry

### CODE REVIEW

**Review by andresilva — APPROVED** [review]

**Comment by bkontur** [comment]
all the pipelines are green, so what is this waiting for `test Expected — Waiting for status to be reported `?

image

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -12,9 +12,39 @@ concurrency:
...
+    name: Extract integration tests from matrix
+    steps:
+      - uses: actions/checkout@v2
+      - id: itest
+        run: |
+          TASKS=$(echo $(cat .github/workflows/integration-tests-matrix.json) | sed 's/ //g' )
+          echo $TASKS
+          echo "itest=$TASKS" >> $GITHUB_OUTPUT
+
+  runtime:
```
Comment (bkontur) [inline .github/workflows/test.yml:41]:
```suggestion
  runtime-test:
```

**Pull request merged**